library/templates/dcp_developments.yml

```diff
@@ -0,0 +1,56 @@
+dataset:
+  name: &name dcp_developments
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://nyc3.digitaloceanspaces.com/edm-publishing/db-developments/main/{{ version }}/output/devdb.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+      - "X_POSSIBLE_NAMES=longitude"
+      - "Y_POSSIBLE_NAMES=latitude"
+    geometry:
+      SRS: EPSG:4326
+      type: POINT
+
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
+      type: POINT
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+      - "GEOMETRY=AS_WKT"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      # DCP Developments Database
+      This is similar to the housing database but includes all Developments (not only residential)
+    url: https://www1.nyc.gov/site/planning/data-maps/open-data/dwn-housing-database.page#housingdevelopmentproject
+    dependents:
+      - dof_shoreline
+      - council_members
+      - dcp_mappluto
+      - doitt_buildingfootprints
+      - doitt_buildingfootprints_historical
+      - doitt_zipcodeboundaries
+      - dcp_cdboundaries
+      - dcp_cb2010
+      - dcp_censustracts
+      - dcp_school_districts
+      - dcp_boroboundaries_wi
+      - dcp_councildistricts
+      - dcp_firecompanies
+      - doe_school_subdistricts
+      - doe_eszones
+      - doe_mszones
+      - dcp_policeprecincts
+      - hpd_hny_units_by_building
+      - dob_jobapplications
+      - dob_permitissuance
+      - dob_cofos
```

Comment:
Question @mbh329 - seems like this is listing the datasets going into `db-developments`, not the downstream dependents on it, no? The latter being the intent of this part of the template?

Reply:
I think each engineer thought of this section a little differently when we being more cognizant of adding "dependents" to the yaml files, perhaps it makes sense to have both though moving forward and renaming this section? That way its a bit easier to track which are the upstream vs downstream dependents